meal-planner/src/App.js

```diff
@@ -51,6 +133,7 @@ export default function App() {
                             <MealSelectMenu
                                 handleMealClick={selectMeal}
                                 weekdayId={id}
+                                handleRecipeClick={handleShowRecipe}
                             />
                         )}
                     </div>
```

Comment:
What is the footer for? 
If there is no need for it, throw it out :)

Reply:
Later ... ah yeah right  - not supposed to do anything with "later in mind" But that's wer the  cart icon for the shopping list is going.